indico/modules/rb_new/controllers/backend.py

```diff
@@ -26,6 +26,7 @@
 from indico.modules.rb_new.schemas import aspect_schema, rooms_schema
 from indico.modules.rb_new.util import search_for_rooms
 from indico.util.string import natural_sort_key
+from indico.web.util import jsonify
```

Comment:
that's not where `jsonify` comes from. it's `from flask import jsonify` (it only works because that module imports it from flask)